### PR TITLE
Configure the GitHub workflows for Stolostron

### PR DESCRIPTION
**Pull request by mprahl**
Signed-off-by: mprahl <mprahl@users.noreply.github.com>

### CODE REVIEW

**Comment by dhaiducek** [comment]
In light of the other repo changes, I'm assuming this PR will be updated also with the common Sonar workflow.

**Comment by mprahl** [comment]
Updated

**Comment by dhaiducek** [comment]
Looks like some upstream references here need to be replaced with `stolostron`?

**Comment by mprahl** [comment]
This is blocked by https://github.com/open-cluster-management-io/governance-policy-framework-addon/pull/3.

**Comment by dhaiducek** [comment]
Missing `go.mod` replacements, for example: https://github.com/stolostron/config-policy-controller/blob/a3fe2d26855da263262e44676ad0a8a6502a1502/go.mod#L104-L105

**Comment by dhaiducek** [comment]
Oops--sorry. Already fixed. 🙂

**Comment by dhaiducek** [comment]
It looks like the Framework tests are blocked because it doesn't know about this repo/image yet to patch it.

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ This controller watches for changes on `Policies` in the cluster namespace on th
 ## Geting started
 
 Go to the
-[Contributing guide](https://github.com/open-cluster-management-io/community/blob/main/sig-policy/contribution-guidelines.md)
+[Contributing guide](https://github.com/stolostron/community/blob/main/sig-policy/contribution-guidelines.md)
```
Comment (dhaiducek) [inline README.md:43]: This doesn't need to be replaced

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ This controller watches for changes on `Policies` in the cluster namespace on th
 ## Geting started
```
Comment (dhaiducek) [inline README.md:40]: This might need to be updated upstream...
```suggestion
## Getting started
```

File: go.mod [hunk 3]
```diff
@@ -72,16 +74,20 @@ require (
...
+	sigs.k8s.io/kube-storage-version-migrator v0.0.4 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
 replace (
 	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220722155217-630584e8d5aa // CVE-2021-43565
 	k8s.io/client-go => k8s.io/client-go v0.23.10
+	open-cluster-management.io/governance-policy-propagator => github.com/stolostron/governance-policy-propagator v0.0.0-20220727212642-86a318ab17cf
+	open-cluster-management.io/multicloud-operators-subscription => github.com/stolostron/multicloud-operators-subscription v1.2.4-0-20211122-7277a37.0.20220727170504-931d9002a21f
```
Comment (dhaiducek) [inline go.mod:92]: Nit: This commit is from July. But I suppose updating it can wait for the PR to update to `1.19`.

**Review by dhaiducek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mprahl) [reply README.md]: Thanks. That was a find/replace error on my part.

File: README.md (hunk 2 above)
Comment (mprahl) [reply README.md]: I included it in an additional commit in this PR:
https://github.com/open-cluster-management-io/governance-policy-framework-addon/pull/3

File: go.mod (hunk 3 above)
Comment (mprahl) [reply go.mod:92]: I just copied what was in status-sync for consistency.

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**